deploy/charts/rook-ceph/templates/cluster-rbac.yaml

```diff
@@ -26,6 +26,11 @@ roles
 ---
 {{ include "library.cluster.roles" . }}
 
+{{- if .Values.monitoring.enabled }}
```

Comment:
@BlaineEXE This is #9383 with `monitoring.enabled` defaulting to false, but you forgot to include the rolebindings.

Reply:
So I did!